### ISSUE TITLE
Add support for boards

### ISSUE THREAD

**Issue by 8utl3r** [Wojnach/vimflowy#8]
When you use boards there isn't a way to jump between the board columns, it just treats the first one as if it were the only one.

**Comment by Wojnach** [comment]
This might be solvable by rewritting the navigational part of the extension to use the workflowy API instead. The current navigational code is quite old and it's based on how the Workflowy page was built years ago. 

I'll take a look at it in the coming weeks. 

In the meantime; one way to jump between board columns is to use the "ctrl-j" EasyMotion feature. Although the letters aren't visible until you hover over the anchor point for that specifc node.

**Comment by 8utl3r** [comment]
Oh crap, I (literally) didn't even see that! Honestly, getting that to show up visibly might be the best fix right there. It kind of follows a nice mental model if lists go up and down but you have to jump columns. It makes intuitive sense.

**Comment by Wojnach** [comment]
yeah hopefully it'll be easy to make the letters visible even with boards. I'd have to inspect the html code to see what they are doing. My implementation of the JumpTo is very simple. I just create a circle filled with same color as the background and then write a letter ontop of it. I suspect the boards has another div layer which is blocking the letter... Or I'm simply injecting the html code at the wrong layer. I dunno, we'll see! :)

**Comment by Wojnach** [comment]
I took a look at the easyJump letter not being visible. I couldn't find a clean way to do it. We either have to manipulate their CSS to make the bullets visible again, during the jump. Or attach the letter to some other div that is visible, but then it has to be placed at an offset to ensure that it doesn't overlap the hidden bullet, which requires us to make assumption of visual design - which is prone to change. I'd rather try to find a different solution that doesn't require us to update the code as soon as they update their CSS. Hopefully there will be a better solution available next time we look at this problem.  

Giving full movement support to boards would be nice, but it requires a rather significant refactor + a lot of testing to ensure that the change doesn't break any existing stuff. I'll get around to it eventually, but it will take some time.
